### DISCUSSION TITLE
Can I turn off update notice

### DISCUSSION THREAD

**Discussion by nateray42** [Difegue/LANraragi#890] [Q&A] [answered]
With the #869 issue, since I am using a large database and also a file watch, I want to stay at 0.8.90.

It is okay to keep myself from juicy new features, but it is sometimes too much when I see endless alarms of 0.9 version updates whenever I return home.

Is there any option to turn off this?

**Comment by Difegue** [comment, marked as answer]
Endless as in, the small topleft notification when you open the index?  

You can enable debug mode to disable update checks, but that'll put a notification about debug mode being enabled instead 🫠
